src/components/map/leaflet-areaselect.js

```diff
@@ -14,13 +14,15 @@ L.AreaSelect = L.Class.extend({
     minHorizontalSpacing: 30,
     minVerticalSpacing: 30,
     keepAspectRatio: false,
+    bbox: null,
   },
 
   initialize: function (options) {
     L.Util.setOptions(this, options);
 
     this._width = this.options.width;
     this._height = this.options.height;
+    this._bbox = this.options.bbox;
```

Comment:
What is this needed for?

Reply:
this was a stray from work on #388 .  I've removed it